aws-kendra-index/src/main/java/software/amazon/kendra/index/DeleteHandler.java

```diff
@@ -68,7 +68,7 @@ public DeleteHandler(Delay delay) {
                 // for more information -> https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-test-contract.html
                 // if target API does not support 'ResourceNotFoundException' then following check is required
                 //.then(progress -> checkForPreDeleteResourceExistence(proxy, proxyClient, request, progress))
```

Comment:
Can we remove this comment? Also, do we know why was this commented out in this first place?

Reply:
Synced offline. Leaving the comments as it is for the reader's understanding.

Reply:
Discussed offline and this is default comment provided by template.